hardware/VirtualThermostat.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include <map>
+#include "ThermostatHardware.h"
+
+typedef std::map<int,  float  > T_Map_LastRoomTemp ;
+
+//duratoiopn of integration window in min
+#define INTEGRAL_DURATION 10
```

Comment:
please do not define anything in headers that are not used in the header.
For instance, put them in the .cpp code file

Reply:
ok done ! this was use in previous version